src/test/java/dk/cngroup/lentils/repository/CypherRepositoryTest.java

```diff
@@ -39,18 +41,18 @@ public void countEmptyDatabaseTest() {
     @Test
     public void addNewCypher() {
         long originalCount = cypherRepository.count();
-        Cypher cypher = new Cypher("Easy", TESTED_STAGE, 49.0988161, 17.7519189, "abc123", "dole");
+        Cypher cypher = new Cypher("Easy", generator.TESTED_STAGE, new Point(49.0988161, 17.7519189), "abc123", "dole");
```

Comment:
Tady radsi prosim pouzij ObjectGenerator.CODEWORD, protoze je konvence, ze kdyz pristupujes ke staticke konstance, tak se pristupuje pres tridu a ne pres instanci. Jinak davam approve, takze si to pak muzes sama mergnout tady tento pullrequest ;)